gallery/src/data-visualization/ReactVisBarGraphPage.tsx

```diff
@@ -0,0 +1,200 @@
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The x position in coordinates of the box to be used. In VerticalBarSeries, if the value provided
+                is a string, this quantity is treated as a category for the x-axis.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>y</NxTableCell>
+              <NxTableCell>VerticalBarSeries: number <br /> HorizontalBarSeries: string/number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The y position in coordinates of the box to be used. In HorizontalBarSeries, if the value provided
+                is a string, this quantity is treated as a category for the y-axis.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <NxH3 className="nx-h3">Basic API Reference</NxH3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>barWidth</NxTableCell>
+              <NxTableCell>number</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                The percentage for which each bar fills the designated bucket. 1.0 means that the bar fills the whole
+                bucket (no padding between bars), while a smaller percentage means more whitespace between the bars.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>className</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                DOM classNames to be added to the wrapper component.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>data</NxTableCell>
+              <NxTableCell>object array</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                Each react-vis chart components require a property, through which
+                we pass an array of objects. These properties correspond to various visual characteristics of the
+                corresponding marks. For example, x and y, which are required for most series types, affect the position
+                of each mark.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>height</NxTableCell>
```

Comment:
Another place where it's confusing to figure out which component is being referenced.